pkg/restapi/operation/operations.go

```diff
@@ -134,13 +136,29 @@ func (c *Operation) createDataVault(rw http.ResponseWriter, config *models.DataV
 		return
 	}
 
-	err := c.vaultCollection.createDataVault(config.ReferenceID)
+	// Creates data vault config store if it does not already exist to store the dataVaultConfiguration
+	err := c.vaultCollection.createConfigVault()
```

Comment:
Optimization: Instead of attempting to create the config store every time the Create Data Vault endpoint is called, you can just create the store once when the EDV server is started.

After that, the only way the store wouldn't exist is if the database admin went in and manually deleted it... but that's true for anything really - I think that type of error is out of the scope of our concern... I think it's fair to assume that someone isn't messing around with the database behind the scenes)